internal/backend/db/db.go

```diff
@@ -161,4 +161,9 @@ type DB interface {
 
 	// -----------------------------------------------------------------------
 	GetOwnership(ctx context.Context, entityID sdktypes.UUID) (string, error)
+
+	// -----------------------------------------------------------------------
+	UpdateValue(ctx context.Context, envID sdktypes.EnvID, key string, f func(sdktypes.Value) (sdktypes.Value, error)) error
```

Comment:
maybe we should prefix it with some string to indicate what are this values
UpdateValue is just too generic

Reply:
i don't follow - you got the `key` there. it's all in the context of a specific key.

Reply:
I mean something like UpdateKeyValValue or something that group this functions as a logical group